src/modules/movieCounter.js

```diff
@@ -0,0 +1,13 @@
+const movieCounter = (count) => {
+  const counter = document.getElementById('counter');
+  count = document.querySelectorAll('.card').length;
+
+  if (count === 0) count = 'No movies';
+  if (count > 999) count = '999+';
+
+  counter.innerText = count;
+  return count;
+};
+
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] Kindly fix the error and remove the linter disabling. Linters are used by all the students to have a standard code and structure, and they help in identifying errors that should be fixed.